From 83425344b2d7339e8f5b0ad230c51278d330d613 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Sat, 22 Jan 2022 11:41:50 +0100
Subject: [PATCH] configure.ac: drop non-standard checks

Not sure what upstream is trying to do here but it does not work
with the latest autoconf.

Upstream-Status: Inactive-Upstream
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 configure.ac | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index aa7dd1c..858db52 100644
--- a/configure.ac
+++ b/configure.ac
@@ -418,7 +418,6 @@ for dir in "" /usr/local $SEARCH_DIR; do
     AC_CHECK_LIB([id3tag -lz], [id3_file_open], [LIBID3TAG_LIBS="-lid3tag -lz"], [unset ac_cv_lib_id3tag_id3_file_open; LDFLAGS="$LDFLAGS_SAVE"; continue])
     break
 done
-test x"$ac_cv_lib_id3tag__lz___id3_file_open" = x"yes" || AC_MSG_ERROR([Could not find libid3tag])
 AC_SUBST(LIBID3TAG_LIBS)
 
 LDFLAGS_SAVE="$LDFLAGS"
@@ -444,10 +443,6 @@ for dir in "" /usr/local $SEARCH_DIR; do
 		  [unset ac_cv_lib_avformat_av_open_input_file; unset ac_cv_lib_avformat_avformat_open_input; LDFLAGS="$LDFLAGS_SAVE"; continue])])
     break
 done
-if test x"$ac_cv_lib_avformat__lavcodec__lavutil__lz___av_open_input_file" != x"yes" &&
-   test x"$ac_cv_lib_avformat__lavcodec__lavutil__lz___avformat_open_input" != x"yes"; then
-   AC_MSG_ERROR([Could not find libavformat - part of ffmpeg])
-fi
 AC_SUBST(LIBAVFORMAT_LIBS)
 
 AC_CHECK_LIB(pthread, pthread_create)
